doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -6,6 +6,12 @@ This API changelog is experimental and we would love feedback on its usefulness.
 .. contents:: |toctitle|
     :local:
     :depth: 1
```

Comment:
Adding a line here will fix this:

`/Users/PDurbin/github/iqss/dataverse/doc/sphinx-guides/source/api/changelog.rst:9:Explicit markup ends without a blank line; unexpected unindent.`